system/settings/settings.xml

```diff
@@ -91,7 +91,7 @@
           <control type="list" format="string" />
         </setting>
       </group>
-      <group id="2">
+      <group id="2" label="371" help="-1">
```

Comment:
I'm not sure has seen that all other stuffs has used it and thought to add it also there. But from my side no help needed for groups, is only to reduce settings label size, I remove it.